CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - **[FIX]** Fix background for `HighlightSection`
 - **[UPDATE]** Rename left nav widget section from 'Others' to 'Widgets'
+- **[UPDATE]** `ItineraryCollapsible` extended by default when 1 stopover
```

Comment:
`- **[UPDATE]** `Itinerary` is no more collapsed when we have only one stopover`